src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__mon">
```

Comment:
you don't need this wrapper

Reply:
that's the only wat I can do this: Display a calendar in the middle of the screen.